src/main/java/com/luv2code/ecommerce/controller/CheckoutController.java

```diff
@@ -40,10 +43,12 @@ public ResponseEntity<String> createPaymentIntent(@RequestBody PaymentInfo payme
 
         logger.info("paymentinfo.amount " + paymentInfo.getAmount());
 
+        // comments
         PaymentIntent paymentIntent = checkoutService.createPaymentIntent(paymentInfo);
 
         String paymentString = paymentIntent.toJson();
 
+        // comments
         return new ResponseEntity<>(paymentString, HttpStatus.OK);
```

Comment:
Prouci Jackson, omogucava ta vracas objekte, predas mu `paymentIntent` i on ce ti sam to sve convertati u json, cak niti netrebas koristiti `ResponseEntity`.